src/startupscriptgenerator/common/buildManifestHelper.go

```diff
@@ -6,42 +6,43 @@
 package common
 
 import (
-    "startupscriptgenerator/common/consts"
-    "github.com/BurntSushi/toml"
-    "log"
-    "path/filepath"
+	"log"
```

Comment:
not sure why this got changed, but you can ignore viewing white space changes in Github if it is too distracting